profile.html

```diff
@@ -29,14 +28,13 @@ <h1>Profile</h1>
                 nulla. Class aptent taciti sociosqu ad litora torquent per conubia nostra,
                 per inceptos himenaeos. </p>
             <h2>Contact:</h2>
-            <p>Email: kob123@hotmail.co.uk</p>
-            <p>Phone: 07880 993667</p>
+            <p class="contact">Email: kob123@hotmail.co.uk</p>
+            <p class="contact">Phone: 07880 993667</p>
```

Comment:
might be worth putting dummy info in contact, this is up to you, but remember this site will be public.

Reply:
good point...